CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ based on http://keepachangelog.com/en/1.0.0/
 
 ### Added
 - Basic spec for the LaTeX grammar.
+- Grammar for `gls` and `glstext` like commands
```

Comment:
Could you add a period at the end of the sentence to make our changelog consistent? I'm afraid we don't document how to document a changelog anywhere, but I'll be very happy if you'd agree to change.

Reply:
Yes of course.